File: 2003/CS101SamplesAll/CS101SamplesWebDevelopment/WebParts/CallCenterRepUserControl.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

public partial class CallCenterRepUserControl_ascx : System.Web.UI.UserControl
{
    // Page events are wired up automatically to methods 
    // with the following names:
    // Page_Load, Page_AbortTransaction, Page_CommitTransaction,
    // Page_DataBinding, Page_Disposed, Page_Error, Page_Init, 
    // Page_Init Complete, Page_Load, Page_LoadComplete, Page_PreInit
    // Page_PreLoad, Page_PreRender, Page_PreRenderComplete, 
    // Page_SaveStateComplete, Page_Unload
     
    protected void Page_Load(object sender, EventArgs e)
    {
    }

	[Personalizable]
	public string Name
	{
		get
		{
			return NameTextbox.Text;
		}
		set
		{
			NameTextbox.Text = value;
		}
	}

	[Personalizable]
	public string EmployeeId
	{
		get
		{
			return EmployeeIdTextbox.Text;
		}
		set
		{
			EmployeeIdTextbox.Text = value;
		}
	}

	[Personalizable]
	public string Extension
	{
		get
		{
			return ExtensionTextbox.Text;
		}
		set
		{
			ExtensionTextbox.Text = value;
		}
	}

	[Personalizable]
	public string DepartmentNumber
	{
		get
		{
			return DepartmentNumberTextbox.Text;
		}
		set
		{
			DepartmentNumberTextbox.Text = value;
		}
	}
}

File: 2003/VB101SamplesAll/VB101SamplesWebDevelopment/WebParts/CallCenterRepUserControl.ascx.vb
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls

Partial Public Class CallCenterRepUserControl_ascx
    Inherits System.Web.UI.UserControl

    ' Page events are wired up automatically to methods 
    ' with the following names:
    ' Page_Load, Page_AbortTransaction, Page_CommitTransaction,
    ' Page_DataBinding, Page_Disposed, Page_Error, Page_Init, 
    ' Page_Init Complete, Page_Load, Page_LoadComplete, Page_PreInit
    ' Page_PreLoad, Page_PreRender, Page_PreRenderComplete, 
    ' Page_SaveStateComplete, Page_Unload

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
    End Sub 'Page_Load

    <Personalizable()> _
    Public Property Name() As String
        Get
            Return NameTextbox.Text
        End Get
        Set(ByVal value As String)
            NameTextbox.Text = value
        End Set
    End Property


    <Personalizable()> _
    Public Property EmployeeId() As String
        Get
            Return EmployeeIdTextbox.Text
        End Get
        Set(ByVal value As String)
            EmployeeIdTextbox.Text = value
        End Set
    End Property


    <Personalizable()> _
    Public Property Extension() As String
        Get
            Return ExtensionTextbox.Text
        End Get
        Set(ByVal value As String)
            ExtensionTextbox.Text = value
        End Set
    End Property


    <Personalizable()> _
    Public Property DepartmentNumber() As String
        Get
            Return DepartmentNumberTextbox.Text
        End Get
        Set(ByVal value As String)
            DepartmentNumberTextbox.Text = value
        End Set
    End Property
End Class 'CallCenterRepUserControl_ascx